incubator/java-spring-boot2/image/Dockerfile-stack

```diff
@@ -7,17 +7,25 @@ RUN  apt-get -qq update \
   && apt-get -qq clean \
   && rm -rf /tmp/* /var/lib/apt/lists/*
 
-COPY ./LICENSE /licenses/
+USER root
 
+COPY ./LICENSE /licenses/
 COPY ./project /project
 COPY ./config /config
 COPY ./mvn-stack-settings.xml /usr/share/maven/conf/settings.xml
 COPY ./mvn-stack-settings.xml /project/mvn-stack-settings.xml
 
+RUN groupadd --gid 1000 java_group \
+ && useradd --uid 1000 --gid java_group --shell /bin/bash --create-home java_user \
+ && mkdir -p /mvn/repository \
+ && chown -R java_user:java_group /project \
```

Comment:
If we're using the `chown` flag, please could you add a `docker-version` constraint on the `requirements` section in the `stack.yaml`. Should be a value of ">= 17.09.0"

Reply:
I'd avoided the chown flag in my first PR. And yea.. I read him the riot act about the same thing in Slack. :-P

Reply:
In the snip above, that is the chown command, not the chown flag on copy...